lib/ArduinoNative/src/Arduino.cpp

```diff
@@ -130,6 +133,12 @@ static const char* PRG_ARG_RADON_ULZER_PORT_DEFAULT = "65432";
 /** Program argument default value of the configuration file. */
 static const char* PRG_ARG_CFG_FILE_DEFAULT = "config/config.json";
 
+/** Program argument default value of the Platoon ID. */
+static const char* PRG_ARG_PLATOON_ID_DEFAULT = "0";

```

Comment:
you can save the pointer by static const char PRG_ARG_CFG_FILE_DEFAULT[] = "config/config.json";

with existing declaration you create a pointer and a C-string with the pointer point to it.
With the proposed change you only have a C-String (4 bytes down without loosing anything :) )